src/common/scrub_types.cc

```diff
@@ -126,19 +146,24 @@ namespace librados {
 
 void inconsistent_obj_wrapper::encode(bufferlist& bl) const
 {
-  ENCODE_START(1, 1, bl);
+  ENCODE_START(2, 2, bl);
```

Comment:
I assume there's something else that will prevent old clients from seeing this encoding?